packages/@react-spectrum/list/src/listview.css

```diff
@@ -23,15 +25,42 @@
   padding: 0;
   background-color: var(--spectrum-table-background-color, var(--spectrum-global-color-gray-50));
   outline: 0;
+  user-select: none;
 
-  [role="row"] {
+  .react-spectrum-ListView-row {
     outline: none;
   }
+
+  /* give first and last items border-radius to match listview container */
```

Comment:
We could probably do isFirstRow and isLastRow classes to make this cleaner, and only do it on the first and last rows. Currently it does it on the first and last in the virtualized view, which is probably ok too. Just another option, not entirely sure if its actually better.

Reply:
Good catch, not sure why I didn't do this in the first place since I did it for TableView haha.

Reply:
does lastItem get applied to the last row while load more is happening?
maybe it'd be best to remove it if we're loading more